xla/tests/convert_test.cc

```diff
@@ -910,8 +948,8 @@ XLA_TEST_F(ConvertTest, ConvertF16F8e4m3fnRoundtrip) {
       {0x1.0p-8, 0x1.0p-8},    // Denormal without rounding
       {0x1.4p-8, 0x1.0p-8},    // Round-to-even down
       {0x1.Cp-8, 0x1.0p-7},    // Round-to-even up
-      {0x1.5p-7, 0x1.4p-7},    // Round-to-nearest down
-      {0x1.3p-7, 0x1.4p-7},    // Round-to-nearest up
+      {0x1.3p-8, 0x1.0p-8},    // Round-to-nearest down
```

Comment:
What happened here?

Reply:
I checked those values by hand, and found it's easier to reason about values with the same exponent.
I believe nothing really changed here, the test is essentially the same.